src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -137,6 +138,20 @@ class ImageBaseViewer extends BaseViewer {
         this.emit('panstart');
     }
 
+    /**
+     * Determines whether discoverability is enabled
+     *
+     * @private
+     * @param {string} discoverabilityType
+     * @return {boolean} value of whether discoverability is enabled for given type
+     */
+    isDiscoverabilityEnabled(discoverabilityType) {
+        const { experiences = {} } = this.options;
+        const canShow = Object.values(experiences).some(experience => experience.canShow);
+
+        return !canShow && !!this.options[discoverabilityType];
```

Comment:
Since we're inside ImageViewer, can we bake the check for `this.options.enableAnnotationsImageDiscoverability` inside this method to avoid having to pass in a string?